include/engine/api/json_factory.hpp

```diff
@@ -48,13 +48,24 @@ template <typename ForwardIter> util::json::String makePolyline(ForwardIter begi
 }
 
 template <typename ForwardIter>
-util::json::Object makeGeoJSONLineString(ForwardIter begin, ForwardIter end)
+util::json::Object makeGeoJSONGeometry(ForwardIter begin, ForwardIter end)
 {
+    auto distance = std::distance(begin, end);
```

Comment:
nitpick: can this be `coordinate_count` ?  `distance` is a pretty overloaded term already.